logstash-core/lib/logstash/pipeline.rb

```diff
@@ -696,10 +701,21 @@ def stalling_threads_info
       .each {|t| t.delete("status") }
   end
 
+  def collect_dlq_stats
+    dlq_metric = @metric.namespace([:stats, :pipelines, pipeline_id.to_s.to_sym, :dlq])
+    enabled = settings.get_value("dead_letter_queue.enable")
+    dlq_metric.gauge(:enabled, enabled)
+    if enabled
+      dlq_metric.gauge(:path, @dlq_writer.get_path.to_absolute_path)
```

Comment:
Should this appear in metrics? I think we have a separate API to show the runtime settings of Logstash (https://www.elastic.co/guide/en/logstash/current/node-info-api.html) that could be used for showing whether or not the queue is enabled, what path it has, etc.